Add possibility to download latest build (#2378)
kyma-project/cli: docs/user/README.md

- Integrated service management.
- Commands providing useful automation.

> ## How-to-install

Reviewer: ```suggestion
## How to Install
```

> [!WARNING]

> > `v3` is still in the prototyping stage. All commands are still considered alpha - use it at your own risk.

Reviewer: ```suggestion
> `v3` is still in the prototyping stage. All commands are still considered alpha. Use it at your own risk.
```

> ### Stable release

Reviewer: ```suggestion
### Stable Release
```

> [!WARNING]

> > The Kyma CLI has not yet been released as a stable version. To test it before the final stage, use nightly or the latest.

Reviewer: ```suggestion
> The Kyma CLI has not yet been released as a stable version. To test it before the final stage, use the nightly or the latest build.
```

> ### Latest build

Reviewer: ```suggestion
### Latest Build
```

> Download latest (stable release or unstable prerelease) v3 build from the [releases](https://github.com/kyma-project/cli/releases) assets.

Reviewer: ```suggestion
Download the latest (unstable pre-release) v3 build from the [releases](https://github.com/kyma-project/cli/releases) assets.
```
There is no stable release yet, so I'd drop the `stable release` wording

To get the latest Kyma CLI for MacOS or Linux, run the following script from the command line:

```sh
curl -sL "https://raw.githubusercontent.com/kyma-project/cli/refs/heads/main/hack/install_cli_latest.sh" | sh -
kyma@v3
```

> ### Nightly build

Reviewer: ```suggestion
### Nightly Build
```
